Makefile

```diff
@@ -374,7 +374,6 @@ DOCKER_BUILD_CI_ARGS ?=
 
 # Image names that will be found in the local podman image registry after build
 # (tags are always VERSION).
-LOCAL_ARO_RP_IMAGE ?= aro
```

Comment:
removing this means we need to notify all devs that new envvars are required to build - right? Should we update env.example?

Reply:
Updated the env.example